.travis.yml

```diff
@@ -1,8 +1,8 @@
-language: scala
+sudo: false
+dist: xenial
```

Comment:
can you add a comment explaining why this is needed?

also isn't `sudo:` redundant/unsupported these days?

Reply:
I want to explicitly specify the build environment, instead of Travis randomly picking one. It's probably true about `sudo`. I can remove that.

Reply:
I'm not saying it's a bad idea, but what's the reason to pin the distribution?  Have you found issues in Travis CI upgrading builds to newer Ubuntu distributions and, if so, what types of issues?

I ask because we probably don't want to micro-manage nor lag in this configuration either.

Reply:
I also vote for minimal config, staying with defaults, unless there's actually an issue. If something breaks with the default config, it's their problem. If we have a non-standard config, it's ours.

Reply:
This is motivated by security as Ubuntu 14.04 LTS reached EOL on April 2019.
Travis CI did make xenial the default
https://blog.travis-ci.com/2019-04-15-xenial-default-build-environment
but sometimes it takes a while for the rollout to reach older repos as these build image may contain breaking changes. 
See for example https://travis-ci.org/scala/scala/jobs/554521880 log where it's using trusty stable.

By picking xenial now, we can migrate our own choice of timing.

Reply:
That's not an issue here.  We're using Xenial by default.  I wonder if the Scala build is using Trusty because the Travis config has `sudo: required` in it?

Reply:
I just wanted to make sure we're using xenial, so if that's the default Travis CI is already using then I'm happy to remove the specification.

Reply:
For what it's worth, scala/scala defaulted to trusty without `sudo` - https://travis-ci.org/scala/scala/jobs/556437704